### PR TITLE
Redraw optimization

### PR DESCRIPTION
**Pull request by bgptr**
After some profiling, I found these issues:
- leaving the app for some time - or refresh it by F5 or close wallet and reopen it - the `GETPEERINFO_ATTEMPT/GETPEERINFO_SUCCESS` requests pile up (**update**: #2901 resolves this part of the issue)
- the `Privacy` and the `SendTransaction` components are redrawn after every `GETPEERINFO_ATTEMPT/GETPEERINFO_SUCCESS` request despite the redux state did not change at all.
- these two factors can increase CPU usage quite high after some time.

This PR fixes this.

### CODE REVIEW

File: app/containers/Wallet.js [hunk 1]
```diff
@@ -41,12 +40,18 @@ class Wallet extends React.Component {
...
     // peers.
-    this.props.setInterval(() => {
-      if (politeiaEnabled) {
-        getPeerInfo();
-      }
+    this.peerInfoIntervalRef = this.props.setInterval(() => {
+      getPeerInfo();
     }, 10000);
   }
+  componentWillUnmount() {
```
Comment (amassarwi) [inline app/containers/Wallet.js:47]: I already did this in https://github.com/decred/decrediton/pull/2901 after refactoring this container to be
functional component, I would recommend merging #2901 then merge this one for the other part which isn't covered in #2901

**Review by amassarwi — COMMENTED** [review]
Thanks for this @bgptr 

I left a general comment regarding the intervals cleanup in the wallet container which was fixed already in #2901 which also refactors the wallet container to be functional, I would suggest merging #2901 and then this one for the other part.

File: app/containers/Wallet.js (hunk 1 above)
Comment (amassarwi) [reply app/containers/Wallet.js]: What do you think guys @vctt94 @alexlyp ?
Would it make sense rebasing 2901 and merge it at this level ? 
Looks like the intervals cleanup is essential performance improvement.

**Comment by bgptr** [comment]
@amassarwi Great, I've reverted the `Wallet.js` so it won't conflict with your PR.

File: app/containers/Wallet.js (hunk 1 above)
Comment (vctt94) [reply app/containers/Wallet.js]: I think that's fine, can you rebase it, so I can review it?

Comment (amassarwi) [reply app/containers/Wallet.js]: Surething, on it now

**Review by vctt94 — APPROVED** [review]
Thanks, it does improve the perfomance on privacy page!

**Review by alexlyp — APPROVED** [review]

**Pull request merged**